app/helpers/application_helper.rb

```diff
@@ -28,4 +28,25 @@ def markdown(text)
       Redcarpet::Markdown.new(renderer, extensions).render(text).html_safe
     end
   end
+
+  # Format create/update times on posts and comments.
+  def timestamp(item)
+    # TODO: make datetime formats configurable.
+    Time::DATE_FORMATS[:ddmmyyyy_hhmm] = '%d %B %Y %H:%M'
+    Time::DATE_FORMATS[:hhmm] = '%H:%M'
+
+    tz = '-08:00'   # Pacific time. TODO: make time zone configurable.
+    latest = item.updated_at.getlocal(tz).to_s(:ddmmyyyy_hhmm)
+    original = if (item.updated_at < item.created_at.end_of_day)
```

Comment:
The calls are right. They are comparing the updated and created times to see if both events happened on the same day. And if so, the original time is shown just as a time (without the date).